tmt/beakerlib.py

```diff
@@ -167,6 +167,18 @@ def fetch(self):
         self.require = tmt.utils.listify(library.get('require', []))
         self.recommend = tmt.utils.listify(library.get('recommend', []))
 
+        # Create a symlink if the library is deep in the structure
+        if os.path.dirname(self.name).lstrip('/'):
+            link = self.name.lstrip('/')
+            path = os.path.join(self.tree.root, os.path.basename(self.name))
+            self.parent.debug(
+                f"Create a '{link}' symlink as the library is deep.")
+            try:
+                os.symlink(link, path)
+            except OSError:
+                self.parent.warn(
+                    f"Unable to create a '{link}' symlink for a deep library.")
```

Comment:
I guess this can happen if the same directory exists on the "root" level, maybe we could mention it?

Reply:
I've included the actual error message which should make things clear:

        warn: Unable to create a 'very/deep/file' symlink for a deep library ([Errno 17] File exists: 'very/deep/file' -> '/var/tmp/tmt/run-095/plan/file/discover/default/libs/test/LICENSE').